drivers/adc-dac/ad74413r/ad74413r.c

```diff
@@ -312,8 +311,8 @@ int ad74413r_set_info(struct ad74413r_desc *desc, uint16_t mode)
 static int ad74413r_scratch_test(struct ad74413r_desc *desc)
 {
 	int ret;
-	uint16_t val;
-	uint16_t test_val = 0x1234;
+	uint32_t val = 0;
```

Comment:
is this explicit initialization required?

Reply:
it was added by mistake, will remove.